doc/data/messages/a/anomalous-backslash-in-string/bad.py

```diff
@@ -1 +1 @@
-string = "\z"  # [anomalous-backslash-in-string]
+string = "\z"  # [syntax-error]
```

Comment:
We should probably move these then? As this will look weird in our documentation.

Reply:
I'm not sure what the best solution is.

Reply:
Perhaps add a `details.rst` explaining that this is because of our CI?